python/dgl/nn/pytorch/explain/gnnexplainer.py

```diff
@@ -12,32 +12,25 @@
 class GNNExplainer(nn.Module):
     r"""GNNExplainer model from `GNNExplainer: Generating Explanations for
     Graph Neural Networks <https://arxiv.org/abs/1903.03894>`__
-
```

Comment:
Let's keep these empty lines to be consistent with the existing practice. Otherwise, the rendered text in the built documentation can become messy.

Reply:
Could you also do so for `HeteroGNNExplainer`?